src/WebJobs.Script/Config/FunctionsHostingConfiguration.cs

```diff
@@ -30,71 +28,63 @@ public FunctionsHostingConfiguration(IEnvironment environment, ILoggerFactory lo
             {
                 _configFilePath = Environment.ExpandEnvironmentVariables(Path.Combine("%ProgramFiles(x86)%", "SiteExtensions", "kudu", "ScmHostingConfigurations.txt"));
             }
+
+            Utility.ExecuteAfterColdStartDelay(_environment, () =>
```

Comment:
Do we want to wait for cold start to happen or we can initialize synchronously? In case of sync initialization "Initialized" and "IsInitialized" should be removed.